lib/pages/posts-page.js

```diff
@@ -10,7 +10,7 @@ export default class PostsPage extends BaseContainer {
 
 	waitForPosts() {
 		const driver = this.driver;
-		const resultsLoadingSelector = By.css( '.posts__list .is-placeholder:not(.post-image)' );
+		const resultsLoadingSelector = By.css( '.post-item__panel:not( .is-placeholder )' );
```

Comment:
Ah, yeah.  I think actually what we want is `.post-item__panel.is-placeholder`, but keep `! present`.  That way if the placeholder *is* present on the page the function will return `false`, which will cause `driver.wait()` to continue holding and retry.  Once the placeholder is gone the function will return `true` and we'll move on.

*However*...there seems to _always_ be a `.is-placeholder` element visible at the bottom of the list, so that test is tricky.  I've got an idea for how to address it that I'm trying to hash out, but it's taking some time.  If I don't get it before you want to merge we can just dummy-out this check with a fake selector and rely on the fast load times to make it irrelevant for the time being